cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -236,7 +234,45 @@ public static KafkaBridgeCluster fromCrd(Reconciliation reconciliation, KafkaBri
 
         return kafkaBridgeCluster;
     }
-    
+
+
+    private static void fromCrdTemplate(final KafkaBridgeCluster kafkaBridgeCluster, final KafkaBridgeSpec spec) {
```

Comment:
extracted method due to checkstyle complaint about the method complexity when adding more `if` statements